src/client.ts

```diff
@@ -50,7 +50,9 @@ export interface CustomAgent {
   load: (options: FingerprintJS.LoadOptions) => Promise<FingerprintJS.Agent>
 }
 
-export interface FpjsSpaOptions extends FpjsClientOptions {
+export interface FpjsSpaOptions
+  extends Omit<FpjsClientOptions, 'loadOptions'>,
```

Comment:
It was a complicated way to make `loadOptions` optional!

Reply:
True 😅. I've simplified it a bit!